services/app-web/src/pages/StateSubmission/New/NewStateSubmissionForm.test.tsx

```diff
@@ -54,6 +54,16 @@ describe('NewStateSubmissionForm', () => {
         const contractType = await screen.findByText('Base contract')
         await userEvent.click(contractType)
 
+        const riskBasedContractFieldSet = screen.getByText(
+            /Is this a risk-based contract/
+        ).parentElement
+        expect(riskBasedContractFieldSet).toBeDefined()
+
+        await userEvent.click(
+            // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
+            within(riskBasedContractFieldSet!).getByLabelText('No')
```

Comment:
Nice thanks for noticing to add the fields to tests that check form completeness on load